Buildings/Experimental/DHC/CentralPlants/Cooling/Plant.mo

```diff
@@ -346,32 +344,11 @@ equation
-<a href=\"modelica://Buildings.Applications.DataCenters.ChillerCooled.Equipment.ElectricChillerParallel\">
-Buildings.Applications.DataCenters.ChillerCooled.Equipment.ElectricChillerParallel</a>. 
-The chilled water bypass loop is controlled to ensure a minimum flow of chilled water is running through the chillers all the time. 
-The condenser water is cooled by two parallel cooling towers with a bypass loop. See 
-<a href=\"modelica://Buildings.Experimental.DHC.CentralPlants.Cooling.Subsystems.CoolingTowerWithBypass\">
-Buildings.Experimental.DHC.CentralPlants.Cooling.Subsystems.CoolingTowerWithBypass</a> 
-for the details of the modeling of the cooling towers. 
-</li>
-<li>
-The chilled water loop is equiped with two parallel variable speed pumps, 
-which are controlled to maitain a use-determined pressure difference setpoint at the demand side. 
-The condenser water pumps are constant speed with prescribed mass flow rates.
-</li>
-<li>
-The plant operates when it receives an <code>on</code> signal from the external control. 
-The staging of the chillers is based on the calculated cooling load. 
-See <a href=\"modelica://Buildings.Experimental.DHC.CentralPlants.Cooling.Controls.ChillerStage\">
-Buildings.Experimental.DHC.CentralPlants.Cooling.Controls.ChillerStage</a> 
-for the detailed control logic.
-</li>
+<li>The cooling is provided by two parallel chillers instantiated from <a href=\"modelica://Buildings.Applications.DataCenters.ChillerCooled.Equipment.ElectricChillerParallel\">Buildings.Applications.DataCenters.ChillerCooled.Equipment.ElectricChillerParallel</a>. The chilled water bypass loop is controlled to ensure a minimum flow of chilled water is running through the chillers all the time. The condenser water is cooled by two parallel cooling towers with a bypass loop. See <a href=\"modelica://Buildings.Experimental.DHC.CentralPlants.Cooling.Subsystems.CoolingTowerWithBypass\">Buildings.Experimental.DHC.CentralPlants.Cooling.Subsystems.CoolingTowerWithBypass</a> for the details of the modeling of the cooling towers. </li>
+<li>The chilled water loop is equiped with two parallel variable speed pumps, which are controlled to maitain a use-determined pressure difference setpoint at the demand side. The condenser water pumps are constant speed with prescribed mass flow rates. </li>
+<li>The plant operates when it receives an <span style=\"font-family: Courier New;\">on</span> signal from the external control. The staging of the chillers is based on the calculated cooling load. See <a href=\"modelica://Buildings.Experimental.DHC.CentralPlants.Cooling.Controls.ChillerStage\">Buildings.Experimental.DHC.CentralPlants.Cooling.Controls.ChillerStage</a> for the detailed control logic. </li>
 </ul>
-<p>
-<img src=\"modelica://Buildings/Resources/Images/Experimental/DHC/CentralPlants/Cooling/Plant.png\" alt=\"System schematics\"/>. 
-</p>
+<p><img src=\"C:/Users/17206/Desktop/plant.tif\"/>. </p>
```

Comment:
Fix path here.

Reply:
Done.

Reply:
Please add alt text for the image like I did before.

Reply:
Done.